engine/src/flutter/impeller/entity/contents/text_contents.cc

```diff
@@ -182,8 +182,8 @@ void TextContents::ComputeVertexData(
       Point uv_size = (atlas_glyph_bounds.GetSize() + Point(1, 1)) / atlas_size;
 
       Point unrounded_glyph_position =
-          basis_transform *
-          (glyph_position.position + scaled_bounds.GetLeftTop());
+          glyph_bounds.GetLeftTop() +
+          (basis_transform * glyph_position.position);
```

Comment:
@jonahwilliams this is this the thing you were talking about the other day where we scale and unscale when calculating this.  That doesn't happen anymore.